.claude/scripts/mcp_common.sh

```
@@ -2185,6 +2186,86 @@ if should_install_server "beads"; then
+    log_with_timestamp "Setting up MCP server: mcp_mail (HTTP: http://127.0.0.1:8765/mcp/)"
+
+    # Check if server already exists
+    if server_already_exists "mcp_mail"; then
+        echo -e "${GREEN}  ✅ Server mcp_mail already exists, skipping installation${NC}"
+        log_with_timestamp "Server mcp_mail already exists, skipping"
+        INSTALL_RESULTS["mcp_mail"]="ALREADY_EXISTS"
+        SUCCESSFUL_INSTALLS=$((SUCCESSFUL_INSTALLS + 1))
+    else
+        # MCP Mail requires the server to be running locally on port 8765
+        # Check if MCP_MAIL_BEARER_TOKEN is set
+        local mcp_mail_token="${MCP_MAIL_BEARER_TOKEN:-}"
```

Shellcheck (0.11.0)

[error] 'local' is only valid in functions.

(SC2168)